ADAL/src/public/ADTelemetry.h

```diff
@@ -76,15 +57,15 @@
 /*!
     Get a singleton instance of ADTelemetry.
  */
-+ (ADTelemetry*)sharedInstance;
++ (nonnull ADTelemetry*)sharedInstance;
 
 /*!
     Register a telemetry dispatcher for receiving telemetry events.
     @param dispatcher            An instance of ADDispatcher implementation.
     @param aggregationRequired   Specifies if telemetry events will be aggregated on client, 
                                  i.e. all events of a single request will be aggregated as one single event.
```

Comment:
The comments for `aggregationRequired` are a little confusing. Why would I want aggregation? It sounds like a good thing until you see that the delegate interface is a bag of properties rather than being structured around an "event" or "events".